app/models/ability.rb

```diff
@@ -0,0 +1,40 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the passed in user here. For example:
+    #
+    user ||= User.new # guest user (not logged in)
+    if user.is?('superadmin')
```

Comment:
noch eine Frage: ich sehe nicht, dass die is?-Funktion im User-Model definiert ist?
Dann müsste der Code hier eigentlich bei Ausführung fehlschlagen